packages/rocketchat-theme/client/imports/components/sidebar/sidebar-item.css

```diff
@@ -243,12 +247,18 @@
 
 		& * {
 			display: inline !important;
+
```

Comment:
Is there any way we can remove this !important statements?

Reply:
No :(